app/controllers/api/discussions_controller.rb

```diff
@@ -2,22 +2,16 @@ class API::DiscussionsController < API::RestfulController
   load_and_authorize_resource only: [:show, :mark_as_read, :set_volume]
   load_resource only: [:create, :update]
 
-  def inbox_by_date
-    load_and_authorize_group if params[:group_id]
+  def discussions_for_dashboard
     @discussions = page_collection inbox_threads
     respond_with_discussions
   end
 
-  def inbox_by_organization
+  def discussions_for_inbox
```

Comment:
Yeah fair enough. Just FYI, my preference is to use 'threads' as much as possible, because I assume the threads of the future will have lots of non-discussion components. But I have no desire to argue the case. Carry on :)